src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/PsBackupProviderManager.cs

```diff
@@ -0,0 +1,82 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ProviderModel
+{
+    public class PsBackupProviderManager
+    {
+        ProviderData providerData;
+
+        public PsBackupProviderManager(Dictionary<string, object> providerParams)
+        {
+            providerData = new ProviderData(providerParams);
+        }
+
+        public PsBackupProviderManager(ProviderData providerDataIn)
+        {
+            providerData = providerDataIn;
+        }
+
+        public IPsBackupProvider GetProviderInstance(ContainerType containerType)
+        {
+            PsBackupProviderTypes providerType = 0;
+
+            switch (containerType)
+            {
+                case ContainerType.AzureVM:
+                    providerType = PsBackupProviderTypes.IaasVm;
+                    break;
+                default:
+                    break;
+            }
+
+            return GetProviderInstance(providerType);
+        }
+
+        public IPsBackupProvider GetProviderInstance(ContainerType containerType, BackupManagementType backupManagementType)
+        {
+            throw new NotImplementedException();
+        }
+
+        public IPsBackupProvider GetProviderInstance(ContainerType containerType, WorkloadType workloadType)
```

Comment:
According to the PM spec, this will be required by the Get-Item cmdlet for DPM, Venus and Mercury scenarios. Currently added a placeholder.
![image](https://cloud.githubusercontent.com/assets/1173819/13490342/07d7eb88-e152-11e5-9774-30081d5d7e64.png)